README.md

```diff
@@ -372,6 +372,30 @@ Specifies the CSS selector for the element to be referenced by the ``aria-descri
 
 If specified, the first matching element is used. See [Accessibility](#Accessibility) for more information.
 
+##### ``width {Number | String}``
+
+This option allows you to control the dialog's width. Default value is `null` (unspecified)
+
+If you provide a number 'px' metric will be used, on the other hand you are able to provide a specific metric using String like `'40%'`
```

Comment:
If you provide a Number, 'px' will be appended. To use a custom metric, use a String, e.g. `'40%'`.